src/NuGet.Indexing/NuGetQuery.cs

```diff
@@ -194,6 +197,11 @@ private static void AuthorClause(BooleanQuery query, Analyzer analyzer, IEnumera
             query.Add(ConstructClauseQuery(analyzer, "Authors", values), occur);
         }
 
+        private static void PackageTypeClause(BooleanQuery query, Analyzer analyzer, IEnumerable<string> values, Occur occur)
```

Comment:
It's actually not used anymore. I removed it.

---
In reply to: [78817495](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r78817495) [](ancestors = 78817495,75626531)